### PR TITLE
run.py: automatically run the "clock" task

### PR DESCRIPTION
**Pull request by smithfarm**
Companion PR to https://github.com/SUSE/ceph/pull/340

(for alignment with upstream)

### CODE REVIEW

File: teuthology/run.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def get_initial_tasks(lock, config, machine_type):
         # upstream changed this to "clock" in f5d9712 but in openstack
         # we have the ability to sync clocks in the user-data script,
         # so we are keeping "clock.check" for now
-        init_tasks.append({'clock.check': None})
+        init_tasks.append({'clock': None})
```
Comment (kshtsk) [inline teuthology/run.py]: I guess the comment above can be dropped.

Comment (smithfarm) [reply teuthology/run.py]: done

**Comment by smithfarm** [comment]
@susebot run teuthology

**Comment by smithfarm** [comment]
@kshtsk Ping

**Pull request merged**